buildPermissions.py

```diff
@@ -0,0 +1,252 @@
+from django.db import IntegrityError, transaction
+from django.contrib.auth.models import Group, Permission
+from django.contrib.contenttypes.models import ContentType
+
+from core.models.CertificationModels import Certification
+from core.models.DepartmentModels import Department
+from core.models.MemberModels import Member, Staffer
+from core.models.GearModels import Gear
+from core.models.TransactionModels import Transaction
+from core.models.QuizModels import Question, Answer
+
+
+"""
+As permissions are created, they are appended here. Then, all these permissions are appended to the next group. That
```

Comment:
I would refactor this to a class instead of a collection of methods

Reply:
But we can leave it for now

Reply:
Would be good. But once we're in production this will never be run, so, eh